README.md

```diff
@@ -1 +1,23 @@
-# coding-standards
\ No newline at end of file
+# coding-standards
+
+### Git
+
+#### The art of committing
+
+ - Commit content
+   - A logically connected change
+   - Must not fail a build
+   - Break up every work into small tasks (no more than few hours of work per commit)
+   - Isolate meaningless commits
+ - Commit message
```

Comment:
Although this is somehow dependent on the size of the commit, I think we should say something about the commit message maximum length too.